catalog/app/containers/Bucket/PackageDirectoryDialog.tsx

```diff
@@ -258,24 +260,52 @@ function DialogForm({
     [editorElement, handleNameChange, setMetaHeight],
   )
 
+  const username = redux.useSelector(authSelectors.username)
+
+  const getDefaultPackageName = React.useCallback(
+    (workflow) =>
+      packageHandle.convert(workflow?.packageHandle, 'files', {
+        directory: s3paths.ensureNoSlash(path),
+        username: PD.getUsernamePrefix(username),
+      }),
+    [path, username],
+  )
+
+  const [initialValues, setInitialValues] = React.useState({
+    files: initialFiles,
```

Comment:
this way files initial value wont be updated on input props change (it shouldnt be a problem in practice bc it wont be changing while the dialog is open, but anyways, we should keep that in mind).
btw why did you consolidate all the initial values into one object and put it into state var?
what are the benefits of this vs using separate computed and state vars?

Reply:
Just one benefit — it works :)
I couldn't make it working with updating `initialValue` per field, they just don't update